algorithms/src/fft/polynomial/dense.rs

```diff
@@ -123,8 +123,7 @@ impl<F: Field> DensePolynomial<F> {
     }
 
     /// Perform a naive n^2 multiplication of `self` by `other`.
-    #[cfg(test)]
-    fn naive_mul(&self, other: &Self) -> Self {
+    pub fn naive_mul(&self, other: &Self) -> Self {
```

Comment:
Why was the `#[cfg(test)]` removed? Is there a case where we use it in prod now?

Reply:
Hm I used it at one point, but seem to have moved to a more efficient algorithm. I'll revert the change.

Reply:
```suggestion
	#[cfg(test)]
    fn naive_mul(&self, other: &Self) -> Self {
```